lemur/certificates/cli.py

```diff
@@ -164,9 +164,11 @@ def request_reissue(certificate, notify, commit):
             new_cert = reissue_certificate(certificate, replace=True)
             print("[+] New certificate named: {0}".format(new_cert.name))
             try:
-                # the notifications should not be able to cause this to fail, so surround with a try
-                if isinstance(new_cert, Certificate) and notify and not new_cert.endpoints:
-                    send_reissue_no_endpoints_notification(new_cert)
+                # The notifications should not be able to cause this to fail, so surround with a try.
+                # Endpoints get moved from old to new cert during rotation, so the new cert won't have endpoints yet;
+                # instead, we have to check the old cert for endpoints.
+                if isinstance(certificate, Certificate) and notify and not certificate.endpoints:
```

Comment:
Can I safely remove the `isinstance` check here? In other words, can I assume `certificate` is a `Certificate`?

Reply:
I would say yes, based on the inputs
https://github.com/Netflix/lemur/blob/c200c4664e32c7801bce6913f3b15446274000eb/lemur/certificates/cli.py#L523-L530